blocks/identity-block/features/account-management/default.jsx

```diff
@@ -94,7 +94,6 @@ function AccountManagement({ customFields }) {
 	const header = phrases.t("identity-block.account-information");
 	const socialProfileHeader = phrases.t("identity-block.connected-accounts");
 
-	console.log(header, socialProfileHeader);
```

Comment:
Okay, then yes those files being flagged by lint and the testing checks makes sense. Part of the reason we re-enabled those checks (they broke with the Node version upgrades) is that code that was not working correctly was getting through the review process.

At the very least a ticket should be put in for your team to fix the tests for the account management block, but fixing it now would be ideal. This block has effectively no test coverage since the assertions are mostly false positives right now.